add morning orientation to js3 day plans (#534)
CodeYourFuture/curriculum: common-content/en/module/js3/diagnose/index.md

+++
title = '🌡️ Diagnose'
headless = true
facilitation = false
emoji= '🧩'
[objectives]
1='Identify areas of weakness'
2='Address misconceptions from prep work'
3='Step through some code and check how it runs'
+++

This is a pairing activity!

Each pair will need to split into navigator and driver.
Volunteers can pair up too - they need to drive though!
Navigators you can read the instructions for this workshop as you get setup

This activity will consist of the following steps:

### 🧑‍💻 Predict ➡️ Explain

Given a program or piece of code, you’ll have to explain what the code currently does. Not what it should do.

### 🔍🐛 Find the bug

> Given a target output/behaviour - trainees can identify a bug in the source

Reviewer: ```suggestion
Given a target output/behaviour - trainees can identify a bug in the source code
```

> code

Reviewer: ```suggestion

```

weird br

### 🪜🧭 Propose a strategy

> Given some problem, you’ll have to think about a strategy for solving the given problem. This doesn’t involve coding but stepping back to think about how you could solve the problem. You might want to talk aloud, draw a flow diagram or write out the steps you’d take to solve the given problem

Reviewer: ```suggestion
Given a problem, you’ll have to think about a strategy for solving it. This doesn’t involve coding but stepping back to think about how you could solve the problem. You might want to talk aloud, draw a flow diagram or write out the steps you’d take in your solution.
```
